packages/stacked_tools/lib/src/commands/delete/delete_view_command.dart

```diff
@@ -36,8 +33,7 @@ class DeleteViewCommand extends Command with ProjectStructureValidator {
     final outputPath = argResults!.rest.length > 1 ? argResults!.rest[1] : null;
```

Comment:
All of this code here is repeated. We always want to aim for DRY code (Don't Repeat Yourself). This can be extracted into an abstract class that has a run implementation and calls into a required implemeting function called purge that takes in the output path.

Reply:
I've extracted that function as purgeTemplate and it seemed to best suit the template service, but now the template service file is nearly 550 lines long. So I think that I should separate the file into 3 parts, namely, renderTemplateService, revertTemplateService, and templateServiceUtils.
After this the create commands will depend on renderTemplateService, delete commands will depend on revertTemplateService and both of these new services will depend on templateServiceUtils.